src/catalogue/category/catalogueCardView.component.tsx

```diff
@@ -511,14 +511,14 @@ function CatalogueCardView() {
           flexDirection="column"
           height={cardViewHeight}
           maxHeight={cardViewHeight}
+          width="100%"
         >
-          <Grid item width="100%" sx={{ flexShrink: 0 }}>
+          <Grid width="100%" sx={{ flexShrink: 0 }}>
             <MRT_TopToolbar table={table} />
           </Grid>
-          <Grid item width="100%" sx={{ flex: 1, overflow: 'auto' }}>
+          <Grid width="100%" sx={{ flex: 1, overflow: 'auto' }}>
```

Comment:
It seems this flex: 1 is what causes the difference fixed by columns={12} below. Unfortunately removing it breaks the overflow when the screen is small. There might be an alternative fix but I think its alright for now.